packages/react-aria-components/docs/ListBox.mdx

```diff
@@ -850,7 +850,7 @@ Use the `renderEmptyState` prop to customize what the `ListBox` will display if
 <ListBox
   aria-label="Search results"
   /*- begin highlight -*/
-  renderEmptyState={() => 'No results found.'}
+  renderEmptyState={({isDropTarget, state, isFocused, isFocusVisible}) => <span style={{color: isDropTarget ? 'red' : 'black'}}>No results found.</span>}
```

Comment:
these are great, though in dark mode, it's a little hard to see
<img width="714" alt="Screenshot 2023-10-10 at 1 59 16 PM" src="https://github.com/adobe/react-spectrum/assets/698229/c7555f2a-dcfd-471f-8afa-a9892eb8e8fd">

Instead, we should use one of the variables available to us in our docs, `--spectrum-alias-text-color`
There is another issue, though: I cannot drop anything on this example, so it never goes to red.

As a result, I think it should be in the example a little further down https://github.com/adobe/react-spectrum/blob/main/packages/react-aria-components/docs/ListBox.mdx#L1101
And instead of changing colors (esp to red which is negative), let's swap the string. Something like
```
isDropTarget ? 'This will not be uploaded' : 'Drop items here'
```

Reply:
Yep, updated docs like that in all of the cases and also added for `GridList` (forgot this one)